tools/rosgraph/src/rosgraph/roslogging.py

```diff
@@ -233,31 +234,32 @@ def __init__(self, colorize=True, stdout=None, stderr=None):
         except ImportError:
             self._get_time = None
             self._is_wallclock = None
+        self._template = string.Template(os.environ.get(
+            'ROSCONSOLE_FORMAT', '[${severity}] [${time}]: ${message}') + '\n')
```

Comment:
In the case where the format string is read from the environment variable I would expect it to not have a trailing newline?